tests/state/test_state_api.py

```diff
@@ -104,3 +105,31 @@ def test_scheduled_jobs(self, scheduler, api_client, do_work, frozen_datetime):
                 "tz": "Europe/Paris",
             }
         ]
+
+    def test_enqueue_message(self, stub_broker, do_work, api_client):
+        data_message = {
```

Comment:
would be nice to check broker.enqueue has been called. Use Mock to verify actor.send_with_options has been called and test the input arguments. https://docs.python.org/3/library/unittest.mock-examples.html#using-mock